euphonic/validate.py

```diff
@@ -97,14 +97,14 @@ def _replace_dim(expected_shape: Tuple[int, ...],
     idx = np.where(np.array(expected_shape) == -1)[0]
     if len(idx) == 0 or len(expected_shape) != len(obj_shape):
         return expected_shape
-    else:
-        expected_shape_replaced = np.array(expected_shape)
-        expected_shape_replaced[idx] = np.array(obj_shape)[idx]
-        return tuple(expected_shape_replaced)
+
+    expected_shape_replaced = np.array(expected_shape)
+    expected_shape_replaced[idx] = np.array(obj_shape)[idx]
+    return tuple(expected_shape_replaced)
 
 
 def _ensure_contiguous_attrs(obj: object, required_attrs: List[str],
-                             opt_attrs: Sequence[str] = ()) -> None:
+                             opt_attrs: Sequence[str] = tuple()) -> None:
```

Comment:
Does linter not like `()`?

Reply:
I think this was a misdiagnosis of another problem. Will revert.